storage/innobase/btr/btr0bulk.cc

```diff
@@ -1050,6 +1050,7 @@ BtrBulk::finish(dberr_t	err)
 
 	ut_ad(!sync_check_iterate(dict_sync_check()));
 
-	ut_ad(err != DB_SUCCESS || btr_validate_index(m_index, NULL, false));
+	ut_ad(err == DB_SUCCESS
+	      || btr_validate_index(m_index, NULL, false) == DB_SUCCESS);
```

Comment:
This was a sloppy debug assertion, basically only asserting that InnoDB will not crash when executing `btr_validate_index()`. Good that we are now tightening it.